addon-test-support/-private/execution_context.js

```diff
@@ -1,5 +1,5 @@
-import { getContext as getIntegrationTestContext } from './helpers';
-import { getContext, visit } from './compatibility';
+import { getRoot } from './helpers';
+import { getContext as getTestContext } from './compatibility';
```

Comment:
surprisingly, `getContext` is able to return a `moduleFor` test context. We don't need to always store it on the page object.